frontend/packages/akr/src/tests/cypress/integration/contact-request/contact_request.spec.ts

```diff
@@ -156,12 +156,24 @@ describe('ContactRequestPage', () => {
 
     onContactRequestPage.blurFieldByLabel(/^viesti/i);
     cy.findByText(/tieto on pakollinen/i).should('be.visible');
-    onContactRequestPage.elements.nextButton().should('be.disabled');
+    onContactRequestPage.isNextDisabled();
 
     onContactRequestPage.pasteToFieldByLabel(/^viesti/i, LONG_TEST_MESSAGE);
     onContactRequestPage.elements.byLabel(/^viesti/i).type('{enter}');
 
     cy.findByText(/teksti on liian pitkä/i).should('be.visible');
-    onContactRequestPage.elements.nextButton().should('be.disabled');
+    onContactRequestPage.isNextDisabled();
+  });
+
+  it('should submit the request only if the privacy statement is accepted', () => {
```

Comment:
Jos haluaa niin voi muuttaa testin nimen alun muotoon "should allow submitting the request ..."

Reply:
juu miksei. vaihdettu.